### PR TITLE
Move remaining storage to separate API servers

### PR DESCRIPTION
**Pull request by deads2k**
This removes the last of the storage mega-method and pushes them into separate API servers.

Only the last commit is unique, the rest have already been lgtm'ed

### CODE REVIEW

File: pkg/cmd/server/origin/reststorage_validation_test.go [hunk 1]
```diff
@@ -42,51 +37,51 @@ var KnownUpdateValidationExceptions = []reflect.Type{
...
+	//			if !validatorExists && !exempted {
+	//				t.Errorf("No validator registered for %v (used by %s/%s).  Register in pkg/api/validation/register.go.", kindType, resource, key)
+	//			}
+	//
+	//			if !validationInfo.UpdateAllowed && !exempted {
+	//				t.Errorf("No validateUpdate method registered for %v (used by %s/%s).  Register in pkg/api/validation/register.go.", kindType, resource, key)
+	//			}
+	//		}
+	//	}
+	//}
```
Comment (soltysh) [inline pkg/cmd/server/origin/reststorage_validation_test.go]: I don't think I like this, do we have a followup/issue to have it fixed for 3.7?

**Comment by soltysh** [comment]
Overall this lgtm, but that commented out test worries me a bit :/

**Comment by deads2k** [comment]
> Overall this lgtm, but that commented out test worries me a bit :/

Turns out we were slowly removing the utility of the test already.  opened https://github.com/openshift/origin/issues/16196

**Pull request merged**